### ISSUE TITLE
Handle CSS Lint "Rollup" errors

### ISSUE THREAD

**Issue by jgable** [jgable/grunt-lesslint#32]
It looks like CSS Lint is reporting some "rollup" errors which aren't associated with files and cause problems when I try to get a source map location for them.

Examples:

> Too many font-size declarations (36), abstraction needed. Checks the number of font-size declarations. (font-sizes)
> 
> Too many floats (57), you're probably using them for layout. Consider using a grid system instead.

Here is the structure:

```
{ type: 'warning',
  rollup: true,
  message: 'Too many floats (57), you\'re probably using them for layout. Consider using a grid system instead.',
  rule: 
   { id: 'floats',
     name: 'Disallow too many floats',
     desc: 'This rule tests if the float property is used too many times',
     browsers: 'All',
     init: [Function] } }
```
